server/game/core/utils/EnumHelpers.ts

```diff
@@ -38,6 +38,17 @@ export const isAttackableLocation = (location: LocationFilter) => {
     }
 };
 
+export const isHidden = (location: LocationFilter) => {
```

Comment:
Very useful, good idea. Could you do a me a small favor and update `Card.initializeForCurrentLocation()` to use this helper for setting the `hiddenForOpponent` property